hoot-core-test/src/test/cpp/hoot/core/io/ServiceHootApiDbReaderTest.cpp

```diff
@@ -265,69 +266,57 @@ class ServiceHootApiDbReaderTest : public CppUnit::TestFixture
     CPPUNIT_ASSERT_EQUAL(0.0, node->getX());
     CPPUNIT_ASSERT_EQUAL(0.0, node->getY());
     CPPUNIT_ASSERT_EQUAL(10.0, node->getCircularError());
-    CPPUNIT_ASSERT_EQUAL(0, node->getTags().size());
-    QString tagValue = node->getTags().get(MetadataTags::HootStatus());
-    CPPUNIT_ASSERT_EQUAL(Status::Unknown1, node->getStatus().getEnum());
-    CPPUNIT_ASSERT_EQUAL(10.0, node->getCircularError());
+    CPPUNIT_ASSERT_EQUAL(1, node->getTags().size());
```

Comment:
By setting the include debug flag above in line `146` the `hoot:id` tag is added, increasing the number of tags from `0` to `1` (and others below here too).  This flag makes the `hoot:status` and `error:circular` "tags" also appear.